README.md

```diff
@@ -0,0 +1,45 @@
+# Google Analytics plugin for Snowplow
+
+[![Build Status][travis-image]][travis] [![License][license-image]][license]
+
+## Overview
+
+[sp-ga-plugin.js](sp-ga-plugin.js) lets you fork the payloads sent to Google Analytics to your
+Snowplow endpoint.
+
+|   **[Devops Guide][devops-guide]**   |   **[Analysts Guide][analysts-guide]**  |   **[Developers Guide][developers-guide]**  |
+|:------------------------------------:|:---------------------------------------:|:-------------------------------------------:|
+|  [![i1][devops-image]][devops-guide] | [![i2][analysts-image]][analysts-guide] | [![i3][developers-image]][developers-guide] |
+
+## Quickstart
+
+You can use the plugin by requiring it and specifying your Snowplow endpoint:
+
+```html
+<script>
+  // usual isogram
+  ga('create', 'UA-XXXXX-Y', 'auto');
+  ga('require', 'spGaPlugin', { endpoint: 'https://d3rkrsqld9gmqf.cloudfront.net/i' });
```

Comment:
Not sure what you mean, the `endpoint` is configuration so it could be specified to anything, that's why I chose the cf collector as an example.